models/quality_measures/intermediate/nqf0053_osteoporosis_management_in_women_who_had_a_fracture/quality_measures__int_nqf0053_numerator.sql

```diff
@@ -0,0 +1,270 @@
+{{ config(
+     enabled = var('quality_measures_enabled',var('claims_enabled',var('clinical_enabled',var('tuva_marts_enabled',False))))
+    | as_bool
+   )
+}}
+
+with denominator as (
+
+    select 
+          patient_id
+        , performance_period_begin
+        , performance_period_end
+        , recorded_date
+        , measure_id
+        , measure_name
+        , measure_version
+    from {{ref('quality_measures__int_nqf0053_denominator')}}
+
+)
+
+, value_sets as (
+
+    select
+          code
+        , code_system
+        , concept_name
+    from {{ ref('quality_measures__value_sets') }}
+    
+)
+
+, osteo_procedure_codes as (
+
+    select
+          code
+        , code_system
+        , concept_name
+    from value_sets
+    where lower(concept_name) in (
+          'bone mineral density test'
+        , 'bone mineral density tests cpt'
+        , 'bone mineral density tests icd10pcs'
+        , 'bone mineral density tests hcpcs'
+        , 'dexa dual energy xray absorptiometry, bone density'
+        , 'central dual energy x-ray absorptiometry (dxa)'
+        , 'spinal densitometry x-ray' 
+        , 'ultrasonography for densitometry' 
+        , 'ct bone density axial'
+        , 'peripheral dual-energy x-ray absorptiometry (dxa)'
+    )
+
+)
+
+, procedures_osteo_related as (
+
+    select
+        patient_id
+      , encounter_id
+      , procedure_date
+      , source_code_type
+      , source_code
+    from {{ref('quality_measures__stg_core__procedure')}} as procs
+    inner join osteo_procedure_codes
+        on procs.source_code = osteo_procedure_codes.code
+        and procs.source_code_type = osteo_procedure_codes.code_system
+
+)
+
+, qualifying_procedures as (
+
+    select procedures_osteo_related.*
```

Comment:
Can you list the columns from this table rather than using `*`? It would make the code much easier to read if users didn't have to keep going back to the original CTE to see what's included in the select.

Reply:
It has been replaced with the list of fields `*` was referring to. Thank you.